docker/sui-node/Dockerfile

```diff
@@ -51,9 +51,5 @@ COPY --from=builder /sui/target/release/sui-node /usr/local/bin
 
 ARG BUILD_DATE
 ARG GIT_REVISION
-# Use jemalloc as memory allocator
-ENV LD_PRELOAD /usr/lib/x86_64-linux-gnu/libjemalloc.so
```

Comment:
Is this removed to get clearer signal, or you have found incompatibilities? In the longer term we probably want to use jemalloc regardless of its profiling feature.

Reply:
Yeah this is just so that we can do it environment by environment, to be safer